### PR TITLE
ADD: TRACER blog post

### PR DESCRIPTION
**Pull request by rcjackson**
Here is the blog post for TRACER that shows LMA/CSAPR2 data overlaid over GOES for two different cases during TRACER.

### CODE REVIEW

File: doc/source/blog_posts/TRACER.ipynb [hunk 1]
```diff
@@ -0,0 +1,2267 @@
+{
```
Comment (mgrover1) [inline doc/source/blog_posts/TRACER.ipynb:1]: <p>Can you shorten the post? Maybe "Blending Observations from TRACER using Py-ART". Also, Can you add a link to the TRACER field campaign website, and a bit of an overview of the objective of the field campaign to put this into context/motivation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (mgrover1) [inline doc/source/blog_posts/TRACER.ipynb:1]: <p>Please add an Imports subheader here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (mgrover1) [inline doc/source/blog_posts/TRACER.ipynb:1]: <p>Can you read directly from Adam's repo? Pandas supports remote reading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-DOE/pyart/pull/1296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (rcjackson) [reply doc/source/blog_posts/TRACER.ipynb:1]: Done!

Comment (rcjackson) [reply doc/source/blog_posts/TRACER.ipynb:1]: I've added this in the new commit.

Comment (rcjackson) [reply doc/source/blog_posts/TRACER.ipynb:1]: I have put in a couple of extra sentences about TRACER here and shortened the post to just one case as well as the title. In addition to the brief introduction I've added a link to the TRACER web page.

Comment (mgrover1) [reply doc/source/blog_posts/TRACER.ipynb:1]: Yay! Thanks :)

**Review by mgrover1 — CHANGES_REQUESTED** [review]
Only suggestion I have now is to make sure there is only one copy of the post (remove the one not in the 2022 directory), and make sure that the tags are included at the bottom of the notebook indicating the author, date, etc.

**Review by mgrover1 — APPROVED** [review]

**Pull request merged**